### DISCUSSION TITLE
Failure to confirm academic status

### DISCUSSION THREAD

**Discussion by EricaEke** [community/community#52567] [GitHub Education]
Hi! How are you?

I wanted to talk about my academic status. I don't know why it keeps on being declined so many times. I've sent various images, both taken and uploaded, as a proof of my academic status, saying that I'm still a student, but it still being declined all the time. If sent a picture of my dated school ID, my enrollment contract for 2023's semester, a screenshot of my student profile, but it's still being declined. 

I'll very much appreciate if you could let me know exactly why it has been getting declined and how can I fix that. 

Thanks,
Erica.

**Comment by samus-aran** [comment]
Hi @EricaEke 👋 

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list. If we ask for these specific pieces of information this is because we have verified them being distributed by the institution applying.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Let us know how you get on 👍

**Comment by EricaEke** [comment]
Hi, good afternooon!

Thanks for your response! I'll try resending the info as soon as I can!


I appreciate you! Have a nice day! 🙂
________________________________
De: Ashley Nicolson ***@***.***>
Enviado: terça-feira, 18 de abril de 2023 10:34
Para: community/community ***@***.***>
Cc: Chinyere Erica Chioma Eke ***@***.***>; Mention ***@***.***>
Assunto: Re: [community/community] Failure to confirm academic status (Discussion #52567)


Hi @EricaEke<https://github.com/EricaEke> 👋

Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list. If we ask for these specific pieces of information this is because we have verified them being distributed by the institution applying.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Let us know how you get on 👍
